src/website/src/pages/ec/components/page-banner/docs/usage.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: Usage
+order: 1
+---
+
+import { Paragraph } from '@ecl/website-components';
+
+<Paragraph size="lead">
+  A page banner displays a prominent message and related optional actions.
+  Banners are displayed on the homepage below the site header of EC sites. It
+  conveys an important message to your audience. Page banners are a good way to
+  channel your visitors to complete a specific task (ex: register to an event,
+  apply for funding... ). The user can choose to ignore them or interact with
+  them at any time.
+</Paragraph>
+
+## Anatomy
+
+Banners should have a focused message and provide users with clear actions to take. They should be placed at the top of a layout or screen. When scrolling, banners typically move with content and scroll off the screen. Only one banner should be shown at a time.
+
+Page banner images have an aspect ratio of 4:1, meaning the width will be always four times the height of the images;
+
+| Name                    | Mandatory | Restrictions                                              |
+| ----------------------- | --------- | --------------------------------------------------------- |
+| Supporting illustration | Optional  | choice of 2 background colours when no image              |
+| Container               | Mandatory | 100-150 characters maximum, including punctuation         |
+| Text                    | Mandatory | 3-word max – 25 characters maximum, including punctuation |
+
+## When to use
+
+- Exclusively on the homepage
+- To communicate messages which are important but don’t require user action.
+- To communicate a change, an error, or an important event.
+- To be considered as part of an overall website messaging strategy.
+
+## When not to use
+
+- Do not use more than one banner at a time.
+- Do not use banners as ‘alerts supporting a feedback mechanism’.
+- Don’t include more than one call to action in a banner.
+
+## DO's
+
+- The supporting illustration is
+
+  - relevant
+  - high quality
+  - emotionally persuasive
+  - the minimum ratio is 4:1
+
+## DON'Ts
```

Comment:
```suggestion
## DON'Ts
  - When there is no clear focus, nothing particular to promote 
  - Use sliders; a user is less likely to click on the image
  - emotionally persuasive
  - the minimum ratio is 4:1
```

Reply:
"Use sliders; a user is less likely to click on the image"  what does it mean?

Reply:
" When there is no clear focus, nothing particular to promote " is clearly a "when not to use" rather than a don't